l2-bridge-ethereum/src/constants.ts

```diff
@@ -53,10 +53,17 @@ export const OPTIMISM_L1ERC20_TOKEN_BRIDGE =
   "0x76943c0d61395d8f2edf9060e1533529cae05de6";
 export const BASE_L1ERC20_TOKEN_BRIDGE =
   "0x9de443AdC5A411E83F1878Ef24C3F52C61571e72";
-export const ZKSYNC_L1ERC20_BRIDGE =
+export const ZKSYNC_L1ERC20_TOKEN_BRIDGE =
```

Comment:
The actual contract is named as it was before this PR

```suggestion
export const ZKSYNC_L1ERC20_BRIDGE =
```